ansible/group_vars/all/vars.yaml

```diff
@@ -2,7 +2,7 @@
 tools_namespace: xdwiwd-tools
 dev_namespace: xdwiwd-dev
 test_namespace: xdwiwd-test
-prod_namespace: xdwiwd-tools
+prod_namespace: xdwiwd-prod
```

Comment:
How was this able to successfully deploy prod the first time this was built? Was it hosting prod in the tools namespace?

Reply:
I think we had a false positive :) :(